### PR TITLE
feat(Popup): add show/hide animations

### PR DESCRIPTION
**Pull request by mnajdova**
This PR is simply adding animations for showing/hiding popups. No customization are allowed at this moment, if we need to we can add them as a follow up

### CODE REVIEW

File: packages/fluentui/react-northstar/src/components/Popup/Popup.tsx [hunk 1]
```diff
@@ -540,7 +545,7 @@ const Popup: React.FC<PopupProps> &
   const element = (
     <>
       {triggerElement}
-      {open && (inline ? contentElement : <PortalInner mountNode={mountNode}>{contentElement}</PortalInner>)}
+      {inline ? contentElement : <PortalInner mountNode={mountNode}>{contentElement}</PortalInner>}
```
Comment (mnajdova) [inline packages/fluentui/react-northstar/src/components/Popup/Popup.tsx:548]: The Animation component is responsible for mounting/removing the element now, so no need for this check

**Comment by mnajdova** [comment]
The `Animation` component currently is not using the `useStyles` hook, but it is using directly the `getStyles` function with all performance flags disabled, which is causing a perf issues (at least for second renderings). This PR will not be merged until that issue is solved.

File: packages/fluentui/e2e/server/app.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const ContentList = () => (
 
 const App = () => (
   <BrowserRouter>
-    <Provider theme={teamsTheme}>
+    <Provider theme={teamsTheme} disableAnimations>
```
Comment (layershifter) [inline packages/fluentui/e2e/server/app.tsx:25]: Let's add a comment why it's done 🙏

File: packages/fluentui/react-northstar/src/components/Popup/Popup.tsx [hunk 3]
```diff
@@ -384,17 +385,17 @@ const Popup: React.FC<PopupProps> &
     );
   };
 
-  const renderPopperChildren = ({ placement, scheduleUpdate }: PopperChildrenProps) => {
+  const renderPopperChildren = classes => ({ placement, scheduleUpdate }: PopperChildrenProps) => {
     const content = renderContent ? renderContent(scheduleUpdate) : props.content;
     const popupContent = Popup.Content.create(content || {}, {
       defaultProps: () =>
         getA11yProps('popup', {
-          ...getContentProps(),
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Popup/Popup.tsx:392]: Was this removed intentionally?

File: packages/fluentui/react-northstar/test/__mocks__/react-transition-group.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+export * from 'react-transition-group';
+export const Transition = props => {
```
Comment (layershifter) [inline packages/fluentui/react-northstar/test/__mocks__/react-transition-group.ts:4]: Are there `TransitionProps` ? Can we use them?

File: packages/fluentui/react-northstar/src/components/Popup/Popup.tsx (hunk 3 above)
Comment (mnajdova) [reply packages/fluentui/react-northstar/src/components/Popup/Popup.tsx:392]: No, good catch!

File: packages/fluentui/e2e/server/app.tsx (hunk 2 above)
Comment (mnajdova) [reply packages/fluentui/e2e/server/app.tsx:26]: Added

**Pull request merged**